main.go

```diff
@@ -77,7 +75,6 @@ func init() {
 
 func main() {
 	readConfig(&cfg)
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Nothing actually uses the random number generation, so it was an unnecessary step. The whole rand package was able to be removed.

But the reason I looked into it was because of a compile warning that rand.Seed() was deprecated in Go 1.20, and "an alternative has been available since Go 1.0"